shared/js/dataDriver.js

```diff
@@ -116,6 +116,15 @@ loop.DataDriver = function() {
       this.update("presence", userId, Object.assign({ isHere }, presence));
     }
 
+    /**
+      * Add tile to table of contents.
+      *
+      * @param {Object} tile  The tile to be stored and received by others.
+      */
+      addTile(tile) {
```

Comment:
This should be placed above `sendTextChatMessage` if we're going with the name `add...` And it should probably be named more explicitly to be a table of content tile as a generic `tile` could mean many things.